### PR TITLE
Add LooksRare marketplace link

### PR DESCRIPTION
**Pull request by jxom**
Fixes PAR-42

## What changed (plus any additional context for devs)

This PR adds a link to LooksRare on the NFT expanded state (only for mainnet NFTs).

### iOS

https://user-images.githubusercontent.com/7336481/191627001-9580b790-4fd6-4f29-919d-7c38f6dab96b.mp4

### Android

https://www.loom.com/share/0781d0682ea44cb1916bc9c179a7df85

## What to test

- On both iOS & Android, head to the NFT expanded state, press the "..." button and then you should see "View in Marketplaces..."

### CODE REVIEW

File: src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx [hunk 1]
```diff
@@ -263,6 +281,18 @@ const UniqueTokenExpandedStateHeader = ({
...
+                ...AssetActions[AssetActionsEnum.download],
+              },
+            ]
+          : []),
+        {
+          ...AssetActions[AssetActionsEnum.copyTokenID],
+          discoverabilityTitle:
+            asset.id.length > 15 ? `${asset.id.slice(0, 15)}...` : asset.id,
+        },
         ...(isSupportedOnRainbowWeb
```
Comment (jxom) [inline src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx:296]: Just moving down the "View on..." menu items here.

File: src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx [hunk 2]
```diff
@@ -274,18 +304,16 @@ const UniqueTokenExpandedStateHeader = ({
...
         },
-        ...(isPhotoDownloadAvailable
+        ...(asset.network === Network.mainnet
           ? [
               {
-                ...AssetActions[AssetActionsEnum.download],
+                menuTitle: lang.t(
+                  'expanded_state.unique_expanded.view_on_marketplace'
+                ),
+                menuItems: [AssetActions.opensea, AssetActions.looksrare],
```
Comment (jxom) [inline src/components/expanded-state/unique-token/UniqueTokenExpandedStateHeader.tsx:313]: This is the only new addition.

**Review by terrysahaidak — APPROVED** [review]

**Review by osdnk — CHANGES_REQUESTED** [review]
Please add android screenshots

**Comment by jxom** [comment]
@osdnk there is an android video in description

**Review by pugson — APPROVED** [review]
looks rare to me 🫡

**Review by skylarbarrera — APPROVED** [review]
👌

**Review by osdnk — APPROVED** [review]

**Review by brunobar79 — APPROVED** [review]

**Pull request merged**